src/features/annotation-thread/AnnotationThread.scss

```diff
@@ -0,0 +1,10 @@
+@import 'src/styles/variables';
+
+$annotationThread-width: 340px;
```

Comment:
Variables at this level are global. Since these values are only used once and only in this file, do they need to be pulled out as variables?